src/Tribe/Tickets_Handler.php

```diff
@@ -1044,7 +1053,7 @@ public function save_global_stock( $post_id ) {
 	 *
 	 */
 	public function save_tickets_order( $post_id ) {
-		if ( ! ( isset( $_POST[ 'tribe-tickets-post-settings' ] ) && wp_verify_nonce( $_POST[ 'tribe-tickets-post-settings' ], 'tribe-tickets-meta-box' ) ) ) {
```

Comment:
@Camwyn Could we add a nonce somewhere that would be appropriate? Or, could we stop this from checking $_POST directly, and instead pass it the relevant data once we have started processing the submissions.